lms/djangoapps/instructor/tests/test_enrollment.py

```diff
@@ -378,7 +378,8 @@ def test_reset_student_attempts(self):
         reset_student_attempts(self.course_key, self.user, msk, requesting_user=self.user)
         self.assertEqual(json.loads(module().state)['attempts'], 0)
 
-    def test_delete_student_attempts(self):
+    @mock.patch('courseware.module_render.SCORE_CHANGED.send')
+    def test_delete_student_attempts(self, _mock_signal):
```

Comment:
@nasthagiri made this change to prevent tests from failing. We have a separate integration test in this file that does not mock the signal and checks subsection grades after we expect the signal to have been sent and handled. This is a unit test for enrollments only, and uses a dummy block that causes failures in the `SCORE_CHANGED` signal handlers. Since this is a unit-level test and we have a separate integration test that uses the same function, I think mocking this is absolutely fine.